src/NHN.DtoContracts/NHN.DtoContracts/Flr/Data/PrimaryHealthCareContract.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Runtime.Serialization;
+using NHN.DtoContracts.Common.en;
+
+namespace NHN.DtoContracts.Flr.Data
+{
+    /// <summary>
+    /// Basisdata for en person.
```

Comment:
eh.. Kanskje noe slikt? Basis data for en fastlegekontrakt i FLR som er tilknyttet et primærhelseteam